src/components/data/data-utils.js

```diff
@@ -373,6 +364,16 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
               href: 'https://support.terra.bio/hc/en-us/articles/360025758392'
             }, ['Click here for more info on the table.']),
             p(['Data will be saved in location: 🇺🇸 ', span({ style: { fontWeight: 'bold' } }, 'US '), '(Terra-managed).'])]),
+        dataProvider.tsvFeatures.needsTypeInput && [div({ style: { padding: '0 0 1rem' } }, 'Table name:  '),
```

Comment:
A [Fragment](https://reactjs.org/docs/fragments.html) (imported from react) would be preferable over an array for wrapping multiple elements.

```suggestion
        dataProvider.tsvFeatures.needsTypeInput && h(Fragment, [
          div({ style: { padding: '0 0 1rem' } }, 'Table name:  '),
```